<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Helados</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Donofrio</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Seleccione el tipo de helado</string>
    </property>
    <widget class="QRadioButton" name="rb_Sandwich">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sandwich</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbFrio_Rico">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Frio Rico</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbBonbones">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bonbones</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_Dolceto">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dolceto</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ingrese cantidad de compra:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Cantidad">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtS">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>451</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Procesar">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>340</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Procesar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Borrar">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>400</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Salir">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>460</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>donofrio.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
